frontend/css/search.less

```diff
@@ -0,0 +1,23 @@
+.search-bar {
+  position: relative;
+  margin-top: 1em;
+  input {
```

Comment:
We can truncate the text otherwise it appears under the magnifying glass icon:
```css
    overflow: hidden;
    text-overflow: ellipsis;
    white-space: nowrap;
    display: initial;
    padding-right: calc(45px + 10px + .5em);
```

And possibly make the search bar wider (on the search results page only) to match the mockup with (inline styles for .search-bar element):
```css
    max-width: 400px;
    width: 45%;
```
or something  along those lines.

The result:
![image](https://github.com/metabrainz/listenbrainz-server/assets/6179856/2665f488-c79b-437e-bf07-81f513567745)